exercises/kindergarten-garden/description.md

```diff
@@ -55,4 +55,4 @@ Then if asked for Alice's plants, it should provide:
 
 While asking for Bob's plants would yield:
 
-- Clover, grass, clover, clover
+- Clover, grass, clover, clover
```

Comment:
What has changed here? Is this an end of file newline issue?

Reply:
The description file should not be changed by this PR.
Try checking out a clean version of this file.